source-firestore/discovery.go

```diff
@@ -241,22 +223,32 @@ func (ds *discoveryState) discoverCollection(ctx context.Context, coll *firestor
 		}
 		numDocuments++
 	}
+
+  resourceJSON, err := json.Marshal(resource{
+          Path:         resourcePath,
+          BackfillMode: backfillModeAsync,
+  })
+  if err != nil {
+          return fmt.Errorf("error serializing resource json: %w", err)
+  }
+  var binding = &pc.DiscoverResponse_Binding{
+          RecommendedName:    pf.Collection(collectionRecommendedName(resourcePath)),
+          ResourceSpecJson:   resourceJSON,
+          DocumentSchemaJson: minimalSchema,
+          KeyPtrs:            []string{documentPath},
+  }
+  ds.shared.Lock()
+  ds.shared.bindings = append(ds.shared.bindings, binding)
+  ds.shared.Unlock()
+
 	return nil
 }
 
 func (ds *discoveryState) handleDocument(ctx context.Context, doc *firestore.DocumentSnapshot) error {
-	// Marshal the document to JSON that we'll send to schema inference
-	var docJSON, err = json.Marshal(doc.Data())
-	if err != nil {
-		return fmt.Errorf("error marshalling document: %w", err)
-	}
-
 	// Map the document path to a resource path like 'users/*/messages' and then send
 	// it to the appropriate schema inference worker.
```

Comment:
Comment here could be updated since there isn't a schema inference worker anymore